### PR TITLE
CKAN 2.9 file type labels

### PR DESCRIPTION
**Pull request by Alexandra-Green**
## What this PR accomplishes

Update the file type labels

## Issue(s) addressed

- update the color of file type labels and remove the pseudo 3D styling to keep them as plain rectangles with rounded corners 
- Font color #000, font weight: 600

## What needs review

- File type labels are in the correct colour and font weight on the dataset search and individual dataset/ministry/group page

### Notes:

- Padding and font sizes/font weight are different on the Figma (I added the Figma to the ticket for visual color reference) but the ticket only asks for the color and font weight changed. This may be something to add to dataset search page, if needed. As this PR only reflects the ticket.

### CODE REVIEW

File: ckanext/ontario_theme/fanstatic/external/ontario_theme.css [hunk 1]
```diff
@@ -483,72 +483,11 @@ div.card p {
...
-  height: 35px;
-  background-position: -383px -62px;
-}
-.format-label[data-format=shp],
-.format-label[data-format*=shp] {
-  width: 31px;
-  height: 35px;
-  background-position: -414px -62px;
+.label {
+  color: #FFFFFF;
```
Comment (KatiRG) [inline ckanext/ontario_theme/fanstatic/external/ontario_theme.css:490]: why is this needed?

**Comment by KatiRG** [comment]
New stylings to apply:
   - make font weight semi-bold
   - make font-size smaller (15px)
   - make rectangle smaller:
   
image

   - add 16px spacing between labels:
   
image

**Comment by Alexandra-Green** [comment]
I've added the `0.4em 1em` padding, added `16px` left margin to the file type labels and the font size is now 15px and semi-bold (`font-weight: 600`).

**Review by KatiRG — CHANGES_REQUESTED** [review]
The linear gradients are still being applied to the label backgrounds:
Screen Shot 2023-03-14 at 09 46 56

**Comment by Alexandra-Green** [comment]
> The linear gradients are still being applied to the label backgrounds:

Those aren't even the right colours, so that is weird. I'll take a look

**Review by KatiRG — APPROVED** [review]
@Alexandra-Green  Sorry, must have been a cache thing!

Can you merge the upgrade branch? Otherwise good to go. Thanks!

**Pull request merged**